tsconfig.json

```diff
@@ -21,7 +22,8 @@
     "declaration": true,
     "paths": {
       "src/*": ["./src/*"]
-    }
+    },
+    "types": ["@emotion/react/types/css-prop"]
```

Comment:
I'm surprised this was necessary, based on Emotion's documentation: 
https://emotion.sh/docs/typescript#emotionreact

This line makes part of the old way of adding the `css prop`, and in that case, I'd have thought a `global.d.ts` insertion would've been a better place for it.
https://emotion.sh/docs/typescript#with-the-old-jsx-transform

Reply:
good point, I think I added this pre-emotion 11 upgrade. might not be necessary to carry it forward